### DISCUSSION TITLE
I'm using Macbook Pro 13 2017, which version is suitable for me?

### DISCUSSION THREAD

**Discussion by will-stone** [will-stone/browserosaurus#435] [Q&A] [answered]
I'm using Macbook Pro 13 2017, which version is suitable for me?
I have tried 15.1.5 but it seems not support

_Originally posted by @tuongpm-teko in https://github.com/will-stone/browserosaurus/issues/425#issuecomment-903268618_

**Comment by will-stone** [comment, marked as answer]
@tuongpm-teko Browserosaurus is currently at version 15.1.3, there isn't a version 15.1.5 (yet). You have an intel machine so download the `x64` zip file.

**Reply by tuongbeo** [reply]
Thank you, sorry for mistyping as well

**Reply by will-stone** [reply]
No worries 🙂
